source/dbo/Views/Power BI Reporting/vwUnmatchedDuplicates.sql

```diff
@@ -10,21 +10,42 @@ WITH NtbsNotifications AS (
 		etsGroupings.GroupId AS EtsGroupId,
 		n.NotificationDate,
 		n.CreationDate,
+		COALESCE(cd.TreatmentStartDate, cd.DiagnosisDate, n.NotificationDate) AS TreatmentStartDate,
+		MAX(te.EventDate) AS LastRecordedTreatmentOutcome,
```

Comment:
I'd call this LastRecordedTreatmentOutcomeDate, as we already have a field in reporting called LastRecordedTreatmentOutcome, which holds the actual outcome (i.e. Completed) rather than the date.